mass_operation_abstract/wizard/mass_operation_wizard_mixin.py

```diff
@@ -44,24 +44,26 @@ def default_get(self, fields):
         operation_description_danger = False
 
         # Compute items quantity
-        active_ids = self.env.context.get("active_ids")
+        active_ids = self._context.get("active_ids")
```

Comment:
never use a private attr if you have a public one (which is more likely to stay). In any case these changes are not required.